index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+          <div class="bar1"> 9:41
+          <img src="/images/Group 4.png" alt="icons"></div>
+          <div class="bar2"> 
+            <h1>TRACY MIRANJA</h1>
+          <img src="/images/Icon - Menu.png" alt="togglenav">
+        </div>
+        <section class="headline">
+            <div class="banner">
+              <h1>Hey There. I’m Tracy</h1>  
+              <h2>I’am a Software Developer</h2>
+              <p>I can help you build a product , feature or website Look
+                 through some of my work and experience! If you like what 
+                 you see and have a project you need coded, don’t hestiate
+                  to contact me.</p>
+                  <div class="social-links">
+                    <a href="#"><img src="/images/Vector2.png"></a>
+                    <a href="#"><img src="/images/Vector3.png"></a>
+                    <a href="#"><img src="/images/Vector16.png"></a>
+                    <a href="#"><img src="/images/Vector5.svg"></a>
+            </div>
```

Comment:
- Good job adding social icons and anchor tags to make them as links 💪🏽 
- I'd like to kindly suggest that we replace the `<div>` with a list tag. The list of icons should use a `ul` tag to group them, and `li` per icon.

Something like this 👇🏽 
```
<ul class="social-links">
    <li><a href="#"><img src="/images/Vector2.png"></a></li>
    <li><a href="#"><img src="/images/Vector3.png"></a></li>
     <li><a href="#"><img src="/images/Vector16.png"></a></li>
     <li><a href="#"><img src="/images/Vector5.svg"></a></li>
  </ul>
```
What do you think about this? 

⚠️ Kindly remember to fix your images by adding the dot so that it is rendered on the browser 👍🏽